lib/host-map.nix

```diff
@@ -5,9 +5,9 @@ stdenv: with stdenv.hostPlatform; {
        if isFreeBSD then "Freebsd" else
        if isOpenBSD then "Openbsd" else
        if isNetBSD  then "Netbsd"  else
-       if isHurd    then "Hurd"    else
```

Comment:
Why stip out Hurd?

Reply:
`isHurd` does not exist.  I guess it should be in `inspect.nix`, but it does not seem to be there.

Reply:
I assume it used to be there ...